layer/lambda_utils/lambda_utils/status_endpoint.py

```diff
@@ -38,7 +39,7 @@ def get_mutable_pipeline() -> Generator[ModuleType, None, None]:
         pipeline.__dict__[k] = v
 
 
-@log_action(log_reference=LogReference.STATUS001)
+@log_action(log_reference=LogReference.STATUS001, errors_only=True)
```

Comment:
Again minor point for consistency - we have given all the other log references specific names however the status ones have remained 01, 02 etc